index.html

```diff
@@ -22,31 +21,272 @@
+                    <img src="img/work4.png" alt="work image">
+                </a>
+            </div>
+            <div class="work1-title">
+                <h2>Multi-Post <br>
+                    Stories<br>
+                </h2>
+            </div>
+            <div class="work1-detail">
+                <ul>
+                    <li><a class="text-decoration">CANOPY</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">Back End Dev</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">2015</a></li>
+                </ul>
+            </div>
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration1 blue-color">HTML</a></li>
+                    <li><a class="text-decoration1 blue-color">CSS</a></li>
+                    <li><a class="text-decoration1 blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+    </section>
+    <section class="about-myself">
+        <div class="about-myself-container">
+            <div class="title">
+                <h2>About Myself</h2>
+            </div>
+            <div class="description">
+                <p>Hello I'm a software developer!. I can help you build a product, feature or website. Look through
+                    some
+                    of my work and experience!. If you like what you see and have a project you need coded, don't
+                    hesitate to
+                    contact
+                    me.
+                </p>
+            </div>
+            <div class="lets-connect">
+                LET'S CONNECT
+            </div>
+            <ul class="icons-socialMedia2">
+                <li class="iconnet iconnetMar">
+                    <a href="https://github.com/pradmon"><img src="./img/github.png" alt="github-icon"></a>
+                </li>
+                <li class="iconnet iconnetMar">
+                    <a href="https://www.linkedin.com/in/jorgepradamoncada/"><img src="./img/linkedin.png"
+                            alt="linkedin-icon"></a>
+                </li>
+                <li class="iconnet iconnetMar">
+                    <a href="https://www.angellist.com/"><img src="./img/angellist.png" alt="angellist-icon"></a>
+                </li>
+                <li class="iconnet iconnetMar">
+                    <a href="https://twitter.com/praduar"><img src="./img/twitter1.png" alt="twitter-icon"></a>
```

Comment:
Kindly use the font, and margin as recommended in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)
<img width="1672" alt="Screenshot 2023-04-29 at 08 03 47" src="https://user-images.githubusercontent.com/48546992/235284667-3f6ed051-c29a-4cb3-91af-98e9d08f2f4c.png">
This is the Figma template you chose
<img width="379" alt="Screenshot 2023-04-29 at 07 58 56" src="https://user-images.githubusercontent.com/48546992/235284690-c634cca2-9c7c-4d77-840c-8fab1b5213bb.png">
See how the items are arranged, with spaces between elements, and also the content does not have a white space.
<img width="545" alt="Screenshot 2023-04-29 at 08 00 28" src="https://user-images.githubusercontent.com/48546992/235284783-81b94246-68d9-485d-a38d-1742c323ae24.png">
Fixing this will give a better look to your page and let the user have a better experience